src/Missings.jl

```diff
@@ -219,17 +219,15 @@ function (f::SpreadMissings{F})(xs...; kwargs...) where {F}
         findex = eachindex(first(vecs))
         @assert all(x -> eachindex(x) == findex, vecs)
 
-
         nonmissingmask = fill(true, length(vecs[1]))
         for v in vecs
-            nonmissingmask .&= .!ismissing(v)
+            nonmissingmask .&= .!ismissing.(v)
         end
-        nonmissinginds = findall(nonmissingmask)
```

Comment:
Why remove this? In general I think it's faster to compute indices manually when there are several columns, as otherwise indexing will have to compute the number of `true` values in the mask once for each indexing operation. Probably worth benchmarking.